### DISCUSSION TITLE
I've made a YT video about this

### DISCUSSION THREAD

**Discussion by Haroon21** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#118] [Show and tell]
Honestly, this app is so amazing and the fact that it is free is awesome so I decided to make a yt video about it so more people can find out about it! You can watch it if you want :)

https://youtu.be/8jaTSi1kL1w

**Comment by MatthiasKerbl** [comment]
Hi Haroon21,
Thank you very much for this awesome video. :)
Is it okay for you if I put the link to your video in our readme?

**Reply by Haroon21** [reply]
Yes, a link to the video would be great, thanks!

**Reply by MatthiasKerbl** [reply]
Hi @Haroon21,

I couldn't find any contact details of you but I want to get in touch with you. Could you contact me at news@apphousekitchen.com?

Kind regards,

Matthias
